backend/pkg/web/handler/users.go

```diff
@@ -0,0 +1,54 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/fastenhealth/fasten-onprem/backend/pkg"
+	"github.com/fastenhealth/fasten-onprem/backend/pkg/database"
+	"github.com/fastenhealth/fasten-onprem/backend/pkg/models"
+	"github.com/gin-gonic/gin"
+)
+
+func GetUsers(c *gin.Context) {
+	RequireAdmin(c)
```

Comment:
I understand that this function will return an error response (and presumably close the http connection), however i would prefer that the function also return an error, so we can short-circuit all subsequent code in this GetUsers function.
Maybe `RequireAdmin(c)` should actually be a `IsAdmin(c)` function. 


 ```suggestion
	err := RequireAdmin(c)
	if err != nil {
		//requireAdmin will return error response. 
		return
	}
```

`IsAdmin` might look like:

 ```suggestion
	if IsAdmin(c) {
		c.JSON(http.StatusUnauthorized, gin.H{"success": false, "error": "Unauthorized"})
		return
	}
```

What do you think?